source/usage-examples/find.txt

```diff
@@ -52,5 +52,5 @@ see the :manual:`MongoDB query operator reference documentation
 API Documentation
 ~~~~~~~~~~~~~~~~~
 
-- `Find() <https://pkg.go.dev/go.mongodb.org/mongo-driver@v1.7.0/mongo#Collection.Find>`_
-- `Cursor <https://pkg.go.dev/go.mongodb.org/mongo-driver@v1.7.0/mongo#Cursor>`_
+- `Find() <https://pkg.go.dev/go.mongodb.org/mongo-driver@v1.7.0/mongo#Collection.Find>`__
+- `Cursor <https://pkg.go.dev/go.mongodb.org/mongo-driver@v1.7.0/mongo#Cursor>`__
```

Comment:
Question: should we always use a double _ for links? I've seen single and double

Reply:
Yea! It should be double, I've gotten snooty errors in the past when using single.